tests/test_search_fetch.py

```
@@ -0,0 +1,161 @@
+        )
+        assert fetch_result["content"]["type"] == "json"
+        assert fetch_result["content"]["value"]["tasks"] == ["test", "ship"]
+
+
+@pytest.mark.asyncio
+async def test_search_fetch_error_conditions(relay_server: RelayHubServer) -> None:
+    """Test error conditions for search and fetch operations."""
+    client = Client(relay_server.mcp, name="error-test-client")
+    async with client:
+        # Test empty query
+        search_result = _unwrap(
```

Ruff (0.13.1)

Local variable `search_result` is assigned to but never used

Remove assignment to unused variable `search_result`

(F841)